### PR TITLE
Adding Cisco CVE-2022-20920

### PR DESCRIPTION
**Pull request by santosomar**

### CODE REVIEW

File: 2022/20xxx/CVE-2022-20920.json [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (cve-team) [inline 2022/20xxx/CVE-2022-20920.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/20xxx/CVE-2022-20920.json:1]: ('Expecting value: line 1 column 1 (char 0)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @santosomar,

A few of your submitted PRs are blocked due to errors in the JSON file. Can you take a look at the responses and submit a new commit? We are also getting requests in regards to why this entry is not in the main list yet.

Thanks,
Anthony
CVE Content Team

**Pull request merged**